GliaWidgets/Theme/Survey/Theme+Survey.swift

```diff
@@ -0,0 +1,112 @@
+import Foundation
+import UIKit
+
+public extension Theme {
+    /// Text style.
+    struct Text {
+        /// Foreground hex color.
+        public var color: String
+        /// Font size.
+        public var fontSize: CGFloat
+        /// Font weight.
```

Comment:
Since we are in the process of making fonts accessible (dynamic type) we need to consider alternative way of providing font style instead or in addition to specifying concrete size, thickness.

Reply:
I made to have consistency with existing code and we will introduce changes/improvements with ADA together.